packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -181,7 +158,6 @@ export const Snackbar: FC<SnackbarProps> = (toast) => {
 
   const containerProps: ViewProps = {
     accessibilityViewIsModal: true, // iOS only
-    tabIndex: 0, // Android only
     // Above props prevent screen reader from tap focusing elements behind the Snackbar
```

Comment:
Think we need to duplicate this comment or otherwise rework slightly for clarity--maybe pull the iOS one out as well for something like:
```
  // Below props prevent screen reader from tap focusing elements behind the Snackbar
  if (isAndroid()) {
    containerProps.tabIndex = 0
  }
  if (isIOS()) {
    containerProps.accessibilityViewIsModal = true
  }
```
?